Clarification regarding the use of SVG in NotificationOptions (#18239)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/notifications/notificationoptions/index.md

- `title`
  - : `string`. The notification's title.
- `iconUrl`{{optional_inline}}

>   - : `string`. A URL pointing to an icon to display in the notification. The URL can be: a data URL, a blob URL, a http or https URL, or the [relative URL of a file within the extension](/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities#relative_urls). When using an SVG image, ensure that the image includes height and width directives, for example, `<svg width="something" height="something"…`. Otherwise, the image does not display.

Reviewer: ```suggestion
  - : `string`. A URL pointing to an icon to display in the notification. The URL can be: a data URL, a blob URL, a http or https URL, or the [relative URL of a file within the extension](/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities#relative_urls). When using an SVG image, ensure that the image includes height and width attributes, for example, `<svg width="96" height="96"…`. Otherwise, the image may not be displayed.
```

- `title`
  - : `string`. The notification's title.
- `iconUrl` {{optional_inline}}
  - : `string`. A URL pointing to an icon to display in the notification. The URL can be: a data URL, a blob URL, a http or https URL, or the [relative URL of a file within the extension](/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities#relative_urls). When using an SVG image, ensure that the image includes height and width attributes, for example, `<svg width="96" height="96"…`. Otherwise, the image may not display.

> - `contextMessage`{{optional_inline}}

Author: ```suggestion
- `contextMessage` {{optional_inline}}
```

- `imageUrl`

>   - : `string`. A URL pointing to an image to use in the notification. The URL can be: a data URL, a blob URL, or the [relative URL](/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities#relative_urls) of a file within the extension. When using an SVG image, ensure that the image includes height and width directives, for example, `<svg width="something" height="something"…`. Otherwise, the image does not display.

Reviewer: ```suggestion
  - : `string`. A URL pointing to an image to use in the notification. The URL can be: a data URL, a blob URL, or the [relative URL](/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities#relative_urls) of a file within the extension. When using an SVG image, ensure that the image includes height and width attributes, for example, `<svg width="96" height="96"…`. Otherwise, the image may not be displayed.
```
